assets/constants.ts

```diff
@@ -77,6 +77,22 @@ export default {
 
   "RTL_LANGUAGE_CODES": ["ar"],
 
+  // Remove this method when translation service is extended.
```

Comment:
Add a TODO with an issue number that clearly describes what should be done. (See other TODOs in the codebase for examples of the formatting.) Assign the issue to @DubeySandeep.

Reply:
Added this. Thanks!